### ISSUE TITLE
Returning to the maps menu without using the "Exit to Maps" option counts as abandoning trimps.

### ISSUE THREAD

**Issue by Riven-Spell** [Trimps/bug-tracker#183]
**Describe the bug**
When attempting a humane run, returning to the maps menu without using the "Exit to Maps" option counts as abandoning trimps.

**To Reproduce**
1. Begin a map with AutoFight off.
2. Complete the map with "Return to World" set.
3. With AutoFight off, return to the world via the automated return mechanism
4. Attempt to open the maps menu.

**Expected behavior**
Trimps should not have been abandoned.

**Environment (please complete the following information):**
 - OS version (Windows 10, 11? Linux?) Linux
 - Game Version (are you playing on the web, or on Steam)? Web
 - If you're playing on the web, which browser are you using? [e.g. Chrome, Firefox, Safari] Firefox

**Comment by SereKabii** [comment]
Going from Maps to World using automated return causes the same group that cleared the map to be taken to World. Autofight Off interrupts all processes regarding Battle the moment your trimps are taken to World, meaning not even their stats show up, but the group is already considered to be in World the moment the mechanism goes off, therefore, going to maps will abandon them.

**Comment by Riven-Spell** [comment]
Right-- that behavior feels incorrect, in combination with the fact that the maps button is half-unaware that you're going to abandon the trimps (double click is still required, but text does not update)
